src/status_im2/contexts/wallet/common/screen_base/create_or_edit_account/style.cljs

```diff
@@ -14,7 +14,8 @@
    :z-index  -1})
 
 (def account-avatar-container
-  {:padding-horizontal 20
```

Comment:
did you verify that the other pages using this create_or_edit component still look correct? I would be hesitant to make adjustments in this file 🤔